code/modules/mob/mob.dm

```diff
@@ -346,9 +346,16 @@ var/global/obj/screen/fuckstat/FUCK = new
 // message is output to anyone who can see, e.g. "The [src] does something!"
 // blind_message (optional) is what blind people will hear e.g. "You hear something!"
 /atom/proc/visible_message(var/message, var/blind_message, var/drugged_message, var/blind_drugged_message)
-	if(world.time>resethearers) sethearing()
+	if(world.time>resethearers)
+		sethearing()
 	for(var/mob/virtualhearer/hearer in viewers(get_turf(src)))
-		if(istype(hearer.attached, /mob))
+		if(istype(hearer.attached, /mob/camera/aiEye))
```

Comment:
These three typecheck blocks are a mess and should be handled by a proc on atoms.